datetime/datetime.go

```diff
@@ -0,0 +1,140 @@
+// Package with support of Tarantool's datetime data type.
+//
+// Datetime data type supported in Tarantool since 2.10.
+//
+// Since: 1.6
```

Comment:
@oleg-jukovec, what is a next version of go-tarantool?

Reply:
I think it will be 1.7.0

Reply:
Updated, pushed.